docs/src/modeling/expressions/functions/README.md

```diff
@@ -45,6 +45,8 @@ interagieren. Über die `execution` sind eine Vielzahl an Funktionen bereit, fol
 | `setVariable(key, value)` | Setzt eine bestimmte Variable im Execution Context         | `${execution.setVariable('meineVariable', 'Mein Wert')}` |
 | `hasVariable(key)`        | Prüft, ob eine Variable im Execution Context vorhanden ist | `${execution.hasVariable('meineVariable')}`              |
 
+### Umgang mit Listen
```

Comment:
Wahrscheinlich macht eine eigene kleine Seite mit "Best Practices Scripting" im Modellierungs-Abschnitt Sinn. Hier unter Expressions (kein Scripting in dem Sinne) und Funktionen (Angebote der Plattform für Scripts & Expressions) passt es nicht wirklich.

Reply:
Habe einen neuen Abschnitt angelegt.